src/main/java/com/developerphil/adbidea/ui/MyDeviceChooser.java

```diff
@@ -112,7 +113,7 @@ public MyDeviceChooser(boolean multipleSelection,
 
     myFacet = facet;
     myFilter = filter;
-    myMinSdkVersion = AndroidModuleInfo.getInstance(facet).getRuntimeMinSdkVersion();
+    myMinSdkVersion = AndroidModuleModel.get(facet).getRuntimeMinSdkVersion();
```

Comment:
I didn't try running it with versions previous to 2.4 preview 7, apparently `AndroidModuleInfo.getInstance` doesn't exist anymore? I'll try checking with some other AS versions to be sure.

If that's the case it would be recommended to either wait for 2.4 stable release or do some backward compatibility magic.